frontend/test/resolution/testCopyElision.cpp

```diff
@@ -181,18 +181,63 @@ static void test4() {
 }
 
 static void test5() {
-  testCopyElision("test5",
+  testCopyElision("test5a",
     R""""(
         proc test(cond: bool) {
           var x:int;
+          var z = x;
           if cond {
             var y = x;
             return;
           }
           x;
         }
     )"""",
-    {"M.test@6"});
+    {"M.test@8"});
```

Comment:
Might be nice to put a comment saying this is the ID of `var y = x`, here and elsewhere below.

The expectation is documented in https://github.com/chapel-lang/chapel/blob/4ede1a8f22cd0debc3b865cba115cc0db6c87fb7/frontend/include/chpl/resolution/copy-elision.h#L32-L51 (although of course this comment and the behavior can be updated as needed).